### ISSUE TITLE
embedded dashboard causes iframe-resizer infinite loop

### ISSUE THREAD

**Issue by snarfed** [metabase/metabase#7903]
hi all! thanks for [recommending iframe-resizer for embedded dashboards](https://www.metabase.com/docs/latest/administration-guide/13-embedding.html#resizing-dashboards-to-fit-their-content), it's a great complement. however, it currently makes the iframe grow without bound in an infinite loop, at least in my setup, and i expect others' too.

the culprit is this CSS snippet:

```css
.Dashboard {
    background-color: #f9fbfc;
    min-height: 100vh;
}
```

the dashboard doc consists of `div.EmbedFrame-header`, `div.Dashboard` (in a container), `div.EmbedFrame-footer`. the `100vh` makes `.Dashboard` the full height of the iframe viewport, and the header and footer add height on top of that, so when iframe-resizer increases its height, the document's layout is always even taller. whee. more details in davidjbradshaw/iframe-resizer#594.

(this also caused a bug in firefox that, when you choose a query filter value, scrolls the dashboard down, which hides the query filters, and since iframe-resizer disables scrollbars by default, you can't get them back.)

i overrode this with `.Dashboard {min-height: auto !important; }` in a stylesheet that i injected with #7639, which fixed this bug, and afaict doesn't cause any other problems.

iframe-resizer 3.5.15. reproed in firefox 60, chrome 67, and safari 11.1.1 on mac os x 10.13.5.

**Comment by bedej** [comment]
Any chance we could get the removal of 100vh pulled into the v0.31 release? This seems to break embedded charts (at least if you haven't bought a license to remove the Metabase logo footer, I'm not sure if the bug shows up if you have a license and that DOM element is missing).

**Comment by gkulasik** [comment]
+1 for this issue to be resolved. Any resizing causes the height to constantly grow. Removing min-height: 100vh fixes the issue and I don't see any negative effects in the UI (when using iFrameResizer).

**Comment by kdoh** [comment]
A fix for this (#9076) just shipped in 0.31.2 so I'm going to close this issue to reflect that. Let us know how it works out.

**Comment by gkulasik** [comment]
Seems to be working, thanks!
